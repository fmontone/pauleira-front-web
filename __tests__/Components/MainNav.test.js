import React from 'react';
import { render, cleanup } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';

import MainNav from '~/components/NavBar';

describe('Components/Header', () => {
  afterEach(() => {
    cleanup();
  });

  it('Should render Main Nav Without Crash', () => {
    const { findByTestId } = render(
      <BrowserRouter>
        <MainNav />
      </BrowserRouter>
    );

    expect(findByTestId('main-nav')).toMatchSnapshot();
  });
});
