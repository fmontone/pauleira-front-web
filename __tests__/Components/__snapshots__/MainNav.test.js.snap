// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components/Header Should render Main Nav Without Crash 1`] = `Promise {}`;
